notebooks/runtime_hello_widget.py

```
@@ -0,0 +1,51 @@
+app = marimo.App(width="medium")
+
+with app.setup:
+    import marimo as mo
+
+    from dr_widget.inline import load_dr_runtime
+
+    NAMES = ["Ada", "Grace", "Alan", "Katherine", "Tim", "World"]
+
+
+@app.cell(hide_code=True)
+def _():
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_`

(ANN202)